sunpy/visualization/tests/test_drawing.py

```diff
@@ -18,6 +17,21 @@ def aia171_test_map():
     return Map(get_test_filepath('aia_171_level1.fits'))
 
 
+@pytest.fixture
+def cropped_aia193_sample_map():
+    import sunpy.data.sample
+    m = sunpy.map.Map(sunpy.data.sample.AIA_193_JUN2012)
+    return m.submap(SkyCoord(Tx=300*u.arcsec, Ty=-450*u.arcsec,frame=m.coordinate_frame),
+                    width=500*u.arcsec,
+                    height=500*u.arcsec)
```

Comment:
How about changing one of these significantly so that the extent is not square?

Reply:
Good idea 😛